django-stubs/template/response.pyi

```diff
@@ -17,7 +17,7 @@ class ContentNotRenderedError(Exception): ...
 class SimpleTemplateResponse(HttpResponse):
     content: Any = ...
     closed: bool
-    cookies: SimpleCookie
+    cookies: SimpleCookie[str]
```

Comment:
Or `bytes`? Can you please double check that?

Reply:
I _think_ it's mean to be `str`.

I also just noticed there are other instances of `SimpleCookie` in the codebase without type parameterts, such as in `test/client.pyi`. Maybe someone more familiar with cookies will come along and fix these types.

Reply:
Yes, it is `str` as per docs: https://docs.djangoproject.com/en/4.0/ref/request-response/#django.http.HttpRequest.COOKIES